src/scripts/update-bluegreen-service-via-task-def.sh

```diff
@@ -18,7 +18,7 @@ echo "Created CodeDeploy deployment: $DEPLOYMENT_ID"
 
 if [ "$ECS_PARAM_VERIFY_REV_DEPLOY" == "1" ]; then
     echo "Waiting for deployment to succeed."
-    if $(aws deploy wait deployment-successful --deployment-id "${DEPLOYMENT_ID}"); then
+    if aws deploy wait deployment-successful --deployment-id "${DEPLOYMENT_ID}" then
```

Comment:
before the `then` I am fairly certain a semicolon will be needed. 
![image](https://user-images.githubusercontent.com/33272306/145592396-7d72ac27-6815-4735-a1f9-da45eb86354f.png)